server/controllers/favoriteRecipe.js

```diff
@@ -38,9 +38,8 @@ export default {
           Favorite
             .create({
               recipeId: req.params.recipeId,
```

Comment:
Why not destructure `recipeId` from req.params since you are using it in more than one place